ext/readline/readline.c

```diff
@@ -305,7 +305,7 @@ PHP_FUNCTION(readline_info)
 			oldval = rl_erase_empty_line;
 			if (value) {
 				convert_to_long_ex(value);
-				rl_erase_empty_line = Z_LVAL_PP(value);
+				rl_erase_empty_line = Z_LVAL_P(*value);
```

Comment:
It was intentional. However just replacing it with `_P(*` will not work, usually different changes are required.